app/controllers/puppetdb_foreman/puppetdb_controller.rb

```diff
@@ -5,9 +5,9 @@ def index
       begin
         uri = URI.parse(Setting[:puppetdb_dashboard_address])
         puppetdb_url, layout = case params[:puppetdb]
-                               when 'd3.v2', 'charts' then ["#{uri.path}#{request.original_fullpath}", false]
-                               when 'v3'              then [request.original_fullpath, false]
-                               else                        ["#{uri.path}/index.html", true]
+                               when 'd3.v2', 'charts'                                                    then ["#{uri.path}#{request.original_fullpath}", false]
```

Comment:
Line is too long. [160/80]<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.